src/cdk-experimental/menu/menu-base.ts

```diff
@@ -28,8 +28,11 @@ import {PointerFocusTracker} from './pointer-focus-tracker';
 
 @Directive({
   host: {
+    '[tabindex]': '_isInline ? (_hasFocus ? -1 : 0) : null',
```

Comment:
Why do we have to change the `tabindex` while the element has focus? Since focus landed on it already, the `tabindex` doesn't matter.

Reply:
Without this, the focus lands on the menu (tabindex=0) which sets its first item to tabindex=0 and then forwards focus to it. If I don't change the tabindex on the parent to -1, a shift+tab refocuses the parent which forwards focus back to the first element. So it winds up acting as a focus trap where you can't shift+tab back out of it

Reply:
I see. Another approach could be something like the chips where we temporarily set the `tabindex` temporarily to -1 when the tab key is pressed. Considering this as resolved.

Reply:
This seems to work well in my testing, but if we find bugs with it that sounds like a good alternative approach to try